src/javascript/app/pages/mb_trade/mb_tradepage.js

```diff
@@ -32,6 +33,7 @@ const MBTradePage = (() => {
     };
 
     const init = () => {
+        Header.displayAccountStatus();
```

Comment:
do we need this? it should ideally have updated the notification in header before needing to call it here